lib/parser.js

```diff
@@ -283,6 +322,10 @@ Parser.prototype.includeFile = function (file, config) {
   context.cwf = file; // current working file
   context.mode = 'include';
 
+  if (config) {
+    context.cwf = config.context || conext.cwf;
```

Comment:
This is confusing (and `conext` is mispelled). The same value is sometimes called `actualFilePath`, `actual`, `alternateSource` and `context`. If we stick to `actualFilePath`, let's rename `filePath` to `targetFilePath`.

Reply:
For consistency, how about
* rename `filePath` to `targetFilePath`
* remove  `alternateSource` and use `element.attribs[ATTRIB_BOILERPLATE_PATH]` directly
* `this.dynamicIncludeSrc.push({from: context.cwf, to: filePath, actual: actualFilePath});` ->  `this.dynamicIncludeSrc.push({from: context.cwf, target: targetFilePath, actualFilePath: actualFilePath});`
* Use `config.targetFilePath` instead

Reply:
> For consistency, how about
> * rename `filePath` to `targetFilePath`

Let's try that.

> * remove  `alternateSource` and use `element.attribs[ATTRIB_BOILERPLATE_PATH]` directly

You can assign a variable named `boilerplateFilePath` in the `catch` block.

> * `this.dynamicIncludeSrc.push({from: context.cwf, to: filePath, actual: actualFilePath});` ->  `this.dynamicIncludeSrc.push({from: context.cwf, target: targetFilePath, actualFilePath: actualFilePath});`

Let's try that.

> * Use `config.targetFilePath` instead

```js
context.cwf = (config && config.targetFilePath) || file; // current working file
```